sass/components/_dialog.scss

```diff
@@ -90,14 +90,15 @@
             border-radius: 0 5px 0 0;
 
             .ic-icon {
-                width: 32px;
-                height: 32px;
-                line-height: 34px;
+                width: 27px;
```

Comment:
I had to change it because the old dimensions added padding from the bottom after removing the fixed block height. After my changes, the cross remained in its place.